app/sablier.go

```diff
@@ -112,7 +122,7 @@ func Start(ctx context.Context, conf config.Config) error {
 		SessionsConfig:  conf.Sessions,
 	}
 
-	go server.Start(ctx, logger, conf.Server, strategy)
+	go server.Start(ctx, logger, conf.Server, strategy, dockerPingFunc)
```

Comment:
The server should not have a provider specific function, instead it should be a provider wide health check function.

Related to my previous comment.